### ISSUE TITLE
Cannot install via pip

### ISSUE THREAD

**Issue by suhara** [ScalaConsultants/Aspect-Based-Sentiment-Analysis#8]
I tried to install the package via pip, which returns the following error.

```
% pip install aspect-based-sentiment-analysis
ERROR: Could not find a version that satisfies the requirement aspect-based-sentiment-analysis (from versions: none)
ERROR: No matching distribution found for aspect-based-sentiment-analysis
```

Any ideas?

**Comment by visini** [comment]
What is your python environment and OS?

**Comment by suhara** [comment]
Thanks for your reply.

Okay, I confirmed that I could install via pip with Python 3.7.4. Does the Python version have to be >= 3.7?

I also tried installing the package on Google Colab and failed. I think the default Python version is 3.6. It would be super helpful if the library is compatible with 3.6, so the user can install on Google Colab

* Failed 

- MacOS with Python 3.6.5 (pyenv)
- AWS Deep Learning AMI with Python 3.6.5 (conda)

* Success

- MacOS with Python 3.7.4 (pyenv)
